MatrixKit/Controllers/MXKRoomViewController.m

```diff
@@ -152,6 +141,21 @@ @interface MXKRoomViewController ()
 
 @property (nonatomic) UIImageView *openedAttachmentImageView;
 
+/**
+ Observe kMXSessionWillLeaveRoomNotification to be notified if the user leaves the current room.
+ */
+@property (nonatomic, weak) id kMXSessionWillLeaveRoomNotificationObserver;
```

Comment:
It's the opportunity to rename this property. Remove "k" prefix as it is not a constant. And start with lower case: `mxSessionWillLeaveRoomNotificationObserver`. Could be interesting to do the same for the two properties below.